src/js/mesos-proto.js

```diff
@@ -0,0 +1,2622 @@
+/*
```

Comment:
i think a loader would be overengineered, as the mesos-protos seem to change very often.

what do you think of adding the current state of the protos to this repo along with a build-task?

Reply:
Works for me, also allows for the creation of type definition along with the _bindings_.

Reply:
well, the type definitions won't help us very much as the out-type for stuff form `toObject` (which we currently need to rely on) is `Record<string, any>`! 😢

i'll add the former stuff anyway. and if you think it's worthwhile to have (not very good IMHO) typings lying around somewhere in case we need them, i can add those as well, see https://github.com/dcos/dcos-ui/pull/4188.